packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -185,14 +211,47 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         />
       </Stack>
     ),
-    [itemHovered, menuTitle, ids.participantItemMenuButton]
+    [itemHovered, strings.menuTitle, ids.participantItemMenuButton]
   );
 
   const onDismissMenu = (): void => {
     setItemHovered(false);
     setMenuHidden(true);
   };
 
+  /* @conditional-compile-remove(one-to-n-calling) */
```

Comment:
Conditional compilation note: Please reduce the amount of conditional compilation in this file by following the pattern from https://github.com/Azure/communication-ui-library/pull/2130

* Add a function to encpasulate string related conditional code

```
function partcipantStateStringTrampoline(state: ParticipantState, strings: ParticipantItemStrings): string { 
  /* @conditional-compile-remove(one-to-n-calling) */
  /* @conditional-compile-remove(PSTN-calls) */
  if (participantState === 'Idle' || participantState === 'Connecting') {
    return strings?.participantStateConnecting;
  } else if (participantState === 'EarlyMedia' || participantState === 'Ringing') {
    return strings?.participantStateRinging;
  } else if (participantState === 'Hold') {
    return strings?.participantStateHold;
  }
   // Always return the default for stable builds.
  return '';
};
```

* Add a function to encapsulate the difference in API for state:
```

// This assumes that `ParticipantState` is defined unconditionally, but exported only for beta builds.

function participantStateTrampoline(props: ParticipantItemProps): ParticipantState {
  /* @conditional-compile-remove(one-to-n-calling) */
  /* @conditional-compile-remove(PSTN-calls) */
  return props.state;
  // On stable builds, where this prop does not exist, the default behavior matches that of 'Connecting' state.
  return 'Connected';
}
```

Then no more conditional compilation anywhere.